app/src/main/java/com/endava/myendava/models/CalendarDay.java

```diff
@@ -0,0 +1,43 @@
+package com.endava.myendava.models;
+
+import java.util.List;
+
+public class CalendarDay {
+    private int dayNr;
```

Comment:
don't use Nr or other shortcuts ... it makes the code more easily to read if you are using full words